common-healing.lic

```diff
@@ -140,10 +140,11 @@ module DRCH
   # are the severity and the values are the list of bleeding wounds.
   def parse_bleeders(health_lines)
     bleeders = Hash.new { |h, k| h[k] = [] }
+    bleeder_line_regex = /^\b(inside\s+)?((l\.|r\.|left|right)\s+)?(head|eye|neck|chest|abdomen|back|arm|hand|leg)\b/
     if health_lines.grep(/^Bleeding|^\s*\bArea\s+Rate\b/).any?
       health_lines
-        .drop_while { |line| /^You have .* poison(?:ed)?|^You feel somewhat tired and seem to be having trouble breathing|^You have a dormant infection|^Your wounds are infected|^Your body is covered in open oozing sores/ =~ line }
-        .take_while { |line| /^\b(inside\s+)?((l\.|r\.|left|right)\s+)?(head|eye|neck|chest|abdomen|back|arm|hand|leg)\b/ =~ line }
+        .drop_while { |line| !(bleeder_line_regex =~ line) }
+        .take_while { |line| bleeder_line_regex =~ line }
         .drop_while { |line| /^Bleeding|^\s*\bArea\s+Rate\b/ =~ line }
```

Comment:
Does this drop_while at the end matter? The only things still left should be bleeder info after the take_while.

Reply:
At one point in the development of this script it was, I'll do some tests to confirm if it's still relevant

Reply:
Does not seem to be needed, I'll take it outm thanks